manoseimas/lobbyists/json_urls.py

```diff
@@ -0,0 +1,10 @@
+from django.conf.urls import patterns, url
+
+from manoseimas.lobbyists import views
+
+urlpatterns = patterns(
+    '',
+    url(r'^lobbyists/$', views.lobbyists_json, name='lobbyists_json'),
```

Comment:
Other urls.py in this project usually make the trailing slash optional (`r'^.../?$'`).